src/mypage/components/MypageSubMenu.tsx

```diff
@@ -2,6 +2,12 @@ import { CONTACT_MESSAGE, WITHDRAWAL_MESSAGE } from "@mypage/constants";
 import { useRouter } from "next/router";
 import { useModal } from "@shared/hook";
 
+interface SubMenu {
+  value: string;
+  isEnd?: boolean;
+  onClick: () => void | ((url: string) => void);
```

Comment:
```suggestion
  onClick: VoidFunction;
```

[Lv2] ((url: string) => void) 형식은 사용되지않는것같은데 추가하신 이유가 있을까요? 없으면 삭제하는게 나아보입니다!